x-pack/test/observability_functional/apps/observability/index.ts

```diff
@@ -23,6 +23,5 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./feature_controls'));
     loadTestFile(require.resolve('./pages/rules_page'));
     loadTestFile(require.resolve('./pages/rule_details_page'));
-    loadTestFile(require.resolve('./pages/alert_details_page'));
```

Comment:
Why is this test removed? Was it intentional?

Reply:
It had only tests about feature flag and alert flyout - if feature flag is disabled, alert page should show 404 and '...' menu should open alert flyout instead of page. Since we will always open alert details page, these tests are not needed. I have deleted `./pages/alert_details_page` file.

Reply:
Ahh, I see.

In the `./pages/alert_details_page.ts`, I see the following comment:
> TODO: Add more test cases regarding the feature flag for:
       - alert details URL from the Action variable
       - alert details button from the alert flyout.

So, maybe instead of checking the feature flag, we can check something meaningful instead of those tests. Up to you if you see there is something that we test here.